src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let result = 0;
+
+  if (days >= 7) {
+    result = days * 40 - 50;
+
+    return result;
+  } else if (days >= 3) {
+    result = days * 40 - 20;
+
+    return result;
+  }
+  result = days * 40;
```

Comment:
return it immediately
also, would be nice to move numbers 20m 40, and 50 to variables with meaningful names